inspire_matcher/api.py

```diff
@@ -58,11 +59,17 @@ def match(record, config=None):
         config = current_app.config['MATCHER_DEFAULT_CONFIGURATION']
 
     try:
-        algorithm, doc_type, index = config['algorithm'], config['doc_type'], config['index']
+        algorithm = config['algorithm']
```

Comment:
I think is not used anywhere

Reply:
function `match` from here is used in `workflows.tasks.matching.py` in Next